templates/planet.constructions.html

```diff
@@ -163,36 +160,36 @@ <h2 class="title is-5 is-marginless">Solar plant</h2>
 					{{ else }}
 					<button formaction="/planet/{{ .Data.Planet.ID }}/solar-plant/upgrade"
 						class="is-paddingless Planet-card__button" {{ if .Data.Timer }}disabled{{ end }}>
-						<span class="card-footer-item">Upgrade</span>
+						<span class="card-footer-item"
+								title="Metal: {{ quantity .Data.SolarProdUpgradeCost.Metal }} / Hydrogen: {{ quantity .Data.SolarProdUpgradeCost.Hydrogen }} / Silica: {{ quantity .Data.SolarProdUpgradeCost.Silica }} / Energy: {{ quantity .Data.SolarProdUpgradeCost.Silica }}">
```

Comment:
Energy has silica behind it.

An maybe we should display the difference in energy consumption from the current level to the next level instead of the total energy consumption of the new level.